### PR TITLE
Release notes for CSE 3.0.1

### PR DESCRIPTION
**Pull request by rocknes**
Adding release notes for CSE 3.0.1

---
This change is [Reviewable](https://reviewable.io/reviews/vmware/container-service-extension/830)

### CODE REVIEW

File: docs/cse3_0/CSE_CONFIG.md [hunk 1]
```diff
@@ -122,6 +134,18 @@ Other properties may be left as is or edited to match site conventions.
 
 For more information on AMQP settings, see the [VCD API documentation on AMQP](https://code.vmware.com/apis/442/vcloud#/doc/doc/types/AmqpSettingsType.html).
 
+<a name="mqtt_section"></a>
+### `mqtt` Section
+
+Staring CSE 3.0.1, CSE will support MQTT message buses for communication with
```
Comment (ltimothy7) [inline docs/cse3_0/CSE_CONFIG.md:140]: "Staring" --> starting

**Review by ltimothy7 — APPROVED** [review]

File: docs/cse3_0/CSE_CONFIG.md [hunk 2]
```diff
@@ -148,11 +172,15 @@ The service section contains properties that define CSE server behavior.
...
-| listeners             | Number of threads that CSE server should use                                                                                                               |
+| listeners             | Number of threads that CSE server should use to communicate with AMQP bus and process requests (Removed in CSE 3.0.1)                                      |
+| processors            | Number of threads that CSE server should use for processing requests (Added in CSE 3.0.1)                                                                  |
 | enforce_authorization | If True, CSE server will use role-based access control, where users without the correct CSE right will not be able to deploy clusters (Added in CSE 1.2.6) |
 | log_wire              | If True, will log all REST calls initiated by CSE to VCD. (Added in CSE 2.5.0)                                                                             |
 | telemetry             | If enabled, will send back anonymized usage data back to VMware (Added in CSE 2.6.0)                                                                       |
 
+Note:
+Staring CSE 3.0.1, the `listeners` field has been renamed to `processors`.
```
Comment (ltimothy7) [inline docs/cse3_0/CSE_CONFIG.md:182]: "Staring" --> "Starting"

**Review by sahithi — APPROVED** [review]
:lgtm:  few minor comments

> Reviewed 6 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/vmware/container-service-extension/830#-:-MLold0y1RI3IyxQP9RF:b29tal4)* status: 4 of 6 files reviewed, 7 unresolved discussions (waiting on @goelaashima, @rocknes, and @sahithi)

___
*[docs/cse3_0/CSE_CONFIG.md, line 19 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLojl7eCGaeeiwetVad:-MLojl7eCGaeeiwetVae:b7f8nq7) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/CSE_CONFIG.md#L19)):*
> ```Markdown
> The output of the command varies slightly depending on the target VCD server api
> version that CSE will be using to talk to VCD (controlled by the flag -v).
> Only supported bus type for api version 33.0 and 34.0 is AMQP. For api version
> ```

Only supported "messaging" bus
___
*[docs/cse3_0/CSE_INSTALL_PREREQUISITES.md, line 57 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLokJtH1CiWgZH6ZE5P:-MLokJtH1CiWgZH6ZE5Q:b-khtw0s) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/CSE_INSTALL_PREREQUISITES.md#L57)):*
> ```Markdown
> 
> <a name="service_account"></a>
> ### CSE User
> ```

The title also can be changed to "CSE Service account"
___
*[docs/cse3_0/CSE_SERVER_MANAGEMENT.md, line 14 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLokbVd3OasqeHuik2X:-MLokbVd3OasqeHuik2Y:b-2bghb1) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/CSE_SERVER_MANAGEMENT.md#L14)):*
> ```Markdown
> Extension (CSE) on vCloud Director (VCD). Users who perform these procedures
> are termed as `Cloud Administrators`, they must have at least the role of
> `CSE Service Role` (if the role is not present then the user need to have
> ```

A hyperlink to CSE User section can be helpful here
___
*[docs/cse3_0/INTRO.md, line 34 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLol3jHD6wxYxAL0cxV:-MLol3jHD6wxYxAL0cxW:bawdqo6) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/INTRO.md#L34)):*
> ```Markdown
> and Kubernetes templates. Cloud administrators are expected to posses
> `CSE Service Role` privileges and as well as experience in VCD administration.
> Note: If the role `CSE SErvice Role` is not available, then cloud administrator
> ```

same here - hyperlink to cse service role can be helpful
___
*[docs/cse3_0/RELEASE_NOTES.md, line 23 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLolJAV1y2fKn6gqqun:-MLolJAWBPQ0M5kEy28-:b-bsj95l) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/RELEASE_NOTES.md#L23)):*
> ```Markdown
> | 3.0.1      | 3.0.1   | 1.0.3  | 10.1, 10.0     | 1.7 with 2.5.1     | Native and Ent-PKS cluster management                                                               |
> | NA         | 3.0.1   | 2.0*   | 10.2           | NA                 | Tkg cluster management only                                                                         |
> ```

Why some of the text is in bold? please check.

**Review by rocknes — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/vmware/container-service-extension/830)* status: 4 of 6 files reviewed, 7 unresolved discussions (waiting on @goelaashima, @ltimothy7, and @sahithi)

___
*[docs/cse3_0/CSE_CONFIG.md, line 19 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLojl7eCGaeeiwetVad:-MLooB1zALCPbIo1vvXF:bb6un1j) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/CSE_CONFIG.md#L19)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

Only supported "messaging" bus
</blockquote></details>

Done
___
*[docs/cse3_0/CSE_CONFIG.md, line 140 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLo0R7C_drMX-7H18bx-r1-140:-MLooJG-0hWOMj9tjuCU:bd5l6mq) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/CSE_CONFIG.md#L140)):*
<details><summary><i>Previously, ltimothy7 wrote…</i></summary><blockquote>

"Staring" --> starting
</blockquote></details>

thanks 
___
*[docs/cse3_0/CSE_CONFIG.md, line 182 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLo0R7C_drMX-7H18bx-r1-182:-MLooKUJ5hiLLAGUUu_v:b-1bjpmn) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/CSE_CONFIG.md#L182)):*
<details><summary><i>Previously, ltimothy7 wrote…</i></summary><blockquote>

"Staring" --> "Starting"
</blockquote></details>

thanks :)
___
*[docs/cse3_0/CSE_INSTALL_PREREQUISITES.md, line 57 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLokJtH1CiWgZH6ZE5P:-MLooReEEhij4MshESdt:bb6un1j) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/CSE_INSTALL_PREREQUISITES.md#L57)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

The title also can be changed to "CSE Service account"
</blockquote></details>

Done
___
*[docs/cse3_0/CSE_SERVER_MANAGEMENT.md, line 14 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLokbVd3OasqeHuik2X:-MLopFfC4oUXwwWUiDXy:bb7f2mf) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/CSE_SERVER_MANAGEMENT.md#L14)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

A hyperlink to CSE User section can be helpful here
</blockquote></details>

done
___
*[docs/cse3_0/INTRO.md, line 34 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLol3jHD6wxYxAL0cxV:-MLoorpg6r7b4rfkJ4yw:bb7f2mf) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/INTRO.md#L34)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

same here - hyperlink to cse service role can be helpful
</blockquote></details>

done
___
*[docs/cse3_0/RELEASE_NOTES.md, line 23 at r1](https://reviewable.io/reviews/vmware/container-service-extension/830#-MLolJAV1y2fKn6gqqun:-MLoovEf4MkjUVeNxoAu:b-h4fnve) ([raw file](https://github.com/vmware/container-service-extension/blob/15dadf6ab66a1892c8a4e661282e44e3d31996a1/docs/cse3_0/RELEASE_NOTES.md#L23)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

Why some of the text is in bold? please check.
</blockquote></details>

Text between two * is considered as bold, Github is looking at multiple lines, MD -> html will not do the same :)

**Pull request merged**